src/views/catalog/CreateFromInstanceTypes/components/BootableVolumeList/BootableVolumeList.tsx

```diff
@@ -151,20 +160,29 @@ const BootableVolumeList: FC<BootableVolumeListProps> = ({
           </Tr>
         </Thead>
         <Tbody>
-          {sortedData?.map((bs) => (
-            <BootableVolumeRow
-              rowData={{
-                bootableVolumeSelectedState: !displayShowAllButton
-                  ? selectedBootableVolumeState
-                  : [selectedBootableVolume, onSelectCreatedVolume],
-                preference: preferencesMap[getLabel(bs, DEFAULT_PREFERENCE_LABEL)],
-                pvcSource: getBootableVolumePVCSource(bs, pvcSources),
-              }}
-              activeColumnIDs={activeColumns?.map((col) => col?.id)}
-              bootableVolume={bs}
-              key={getName(bs)}
-            />
-          ))}
+          {sortedData ? (
+            sortedData.map((bs) => (
+              <BootableVolumeRow
+                rowData={{
+                  bootableVolumeSelectedState: !displayShowAllButton
+                    ? selectedBootableVolumeState
+                    : [selectedBootableVolume, onSelectCreatedVolume],
+                  preference: preferencesMap[getLabel(bs, DEFAULT_PREFERENCE_LABEL)],
+                  pvcSource: getBootableVolumePVCSource(bs, pvcSources),
+                }}
+                activeColumnIDs={activeColumns?.map((col) => col?.id)}
+                bootableVolume={bs}
+                key={getName(bs)}
+              />
+            ))
+          ) : (
+            <EmptyState>
+              <Trans ns="plugin__kubevirt-plugin" t={t}>
+                <Title headingLevel="h3">No volumes found</Title>
+                Click the <b> Add volume</b> button to add a volume to boot from
+              </Trans>
+            </EmptyState>
```

Comment:
instead of putting this empty state here I would make before the return statment the following:
if(isEmpty(bootableVolumes) && loaded) return <EmptyState>....;
This way when there are no bootable volumes we hide the pagination and filters as well, which is better as they are not needed if there are not volumes

Reply:
Actually, I've realized that the design doc really doesn't show filtering, but it does show pagination and project name, plus it shows names of the columns. I have no idea why and also how to achieve this expected look :/

Reply:
@avivtur and WDYT about project name? should we display it for empty state or not? And what about names of the columns?

Reply:
if there's no table I think all of those are not needed, including pagination and project name, I think you tell that to Yifat she would agree with you :)

Reply:
So it was decided to hide everything but the project name. It was discussed with Yifat and also with Ronen. The update is in the design doc already. I'm gonna try to implement this decision asap.